src/vs/workbench/services/issue/browser/issueTroubleshoot.ts

```diff
@@ -204,11 +204,11 @@ class TroubleshootIssueService extends Disposable implements ITroubleshootIssueS
 	private askToReproduceIssue(message: string): Promise<TroubleShootResult> {
 		return new Promise((c, e) => {
 			const goodPrompt: IPromptChoice = {
-				label: localize('I cannot reproduce', "I can't reproduce"),
+				label: localize('I cannot reproduce', "I can't Reproduce"),
```

Comment:
This seems weird. I think it should either be "I Can't Reproduce" or "I can't reproduce". I prefer the second. I think we only capitalize the first letter of each word if it's a command.

Reply:
@bpasero This is suggested by @bpasero. I also liked **I can't reproduce**.

Reply:
This is for buttons where we have consistent title casing:

![image](https://github.com/microsoft/vscode/assets/900690/9b21b0c5-6d94-4c16-a71f-8e9e30541ccb)

Using https://titlecaseconverter.com/ shows that `can't` and `cannot` should also be uppercase 👍

Reply:
Wow, there is tool for this. Thanks.

Updated.